README.md

```diff
@@ -1,2 +1,107 @@
-# eslint-plugin-logdna
+# `eslint-plugin-logdna`
 
+## Installation
+
+**Requires eslint also**
+
+We do not use peer dependencies, so make sure that `eslint` is also installed as a dev dependency.
+
+```shell
+npm install eslint-plugin-logdna eslint --save-dev
+```
+
+## Usage
+
+Add `logdna` to the plugins section of your ESLint configuration. You can omit 
+the `eslint-plugin-` prefix. Then you can configure the rules you want to use:
+
+```json
+{
+  "plugins": [
+    "logdna"
+  ],
+  "rules": {
+    "logdna/grouped-require": "error"
+  }
+}
+```
+
+## Rules
+
+### `logdna/grouped-require`
+
+Enforce sorted require declarations within modules
+
+```js
+// Bad
+const foo = require('./lib/foo.js')
+const http = require('http')
+
+// Good
+const http = require('http')
+const foo = require('./lib/foo.js')
+```
+
+#### Options
+
+* `typeOrder` [`<Array>`][] - sort order of require types 
+(default: `['static', 'builtin', 'contrib', 'scoped', 'local']`)
```

Comment:
Perhaps I'd give a full example using them all for clarity.  It's not immediately clear what they all mean.

Reply:
good call, added a full example with comments